prog/rand.go

```diff
@@ -71,6 +71,13 @@ func (r *randGen) randInt() uintptr {
 	return v
 }
 
+func (r *randGen) randRangeInt(begin uintptr, end uintptr) uintptr {
+	v := r.randInt()
+	intRange := end - begin
+	offset := v % intRange
+	return begin + offset
```

Comment:
How to generate inclusive range?  begin + r.rand(end - begin) does not include end.